tests/checktestmatrix/check_test.go

```diff
@@ -0,0 +1,137 @@
+package checktestmatrix
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/cenkalti/backoff/v4"
+	parser "github.com/craigpastro/openfga-dsl-parser"
+	"github.com/openfga/openfga/cmd"
+	"github.com/openfga/openfga/pkg/testfixtures/storage"
+	tupleUtils "github.com/openfga/openfga/pkg/tuple"
+	"github.com/stretchr/testify/require"
+	pb "go.buf.build/openfga/go/openfga/api/openfga/v1"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"gopkg.in/yaml.v2"
+)
+
+type checkTests struct {
+	Tests []checkTest
+}
+
+type checkTest struct {
+	Name       string
+	Model      string
+	Tuples     []tuple
+	Assertions []assertion
+}
+
+type tuple struct {
```

Comment:
Instead of introducing another type called `tuple` can we please use the `*openfgapb.TupleKey`.

Reply:
Oh yeah. Why did I do that? I think I was thinking about breaking up the tuples to be more fine-grained, but I guess that doesn't have to happen now and independently.

Fixed. Thanks!